AppDashboard/templates/shared/navigation.html

```diff
@@ -5,6 +5,15 @@
                 <a id="appscale_logo" href="/"><img src="/images/appscale-logo.png?v=2"/></a>
               <div class="container-fluid">
                 <div class="nav-no-collapse">
+                    <a href="mailto:support@appscale.com" class="btn btn-primary btn-large">Send Feedback</a>
+                    &nbsp;
+                    <a href="https://github.com/AppScale/appscale/issues" class="btn btn-info btn-large" target="_blank">Submit Issue</a>
```

Comment:
This button should link to: https://www.appscale.com/resources/tutorials/redmine